Mathlib/Topology/Algebra/Valued/NormedValued.lean

```diff
@@ -126,4 +139,25 @@ def toNormedField : NormedField L :=
         exact ⟨fun a b hab => lt_of_lt_of_le hab (min_le_left _ _), fun a b hab =>
             lt_of_lt_of_le hab (min_le_right _ _)⟩ }
 
+-- When a field is valued, one inherits a `NormedField`.
+-- Scoped instance to avoid a typeclass loop or non-defeq topology or norms.
+scoped[NormedField] attribute [instance] Valued.toNormedField
```

Comment:
Should this be `scoped[Valued] attribute [instance] Valued.toNormedField`?

Reply:
You had asked
> If it does not cause problems with inference loops, could you also add the reverse NormedField.toValued local instance inside the NormedField namespace?

That's what I did. Can change it to scope different. What do you prefer?

Reply:
I am sorry if I did not explain myself well. This comment is related to my question [above](https://github.com/leanprover-community/mathlib4/pull/17561#issuecomment-2416163020) (which you may have missed). My reason for asking was that, in Lean 3, the local instance you had in the previous commit:

```lean
local instance : NormedField L := Valued.toNormedField L Γ₀
```
could not be added, since that caused inference loops.
In theory, these simple loops can be handled by Lean 4, but in practice there can still be some issues (which is why I was asking whether you had run into any trouble).
From my point of view, `Valued.toNormedField` and `NormedField.toValued` are completely symmetrical, which is why I was asking you to add the second one (but I did not mean you had to modify the first). 
I am not sure whether they both need to be scoped to avoid loops or not (hence my question), but if they do, I think they should live in different scopes.

Reply:
Got it! Yes, I did see the question. Unscoped, existing heartbeat tests for finding instances get worse, so even if there isn't a stuck loop, it worsens performance to make both available globally.

I agree that it makes sense to have both scoped. What scope do you want for each? 
`Valued.toNormedField` in `NormedField` or in `Valued`? And of course, the other one would be the opposite.